src/routes/safe/components/Settings/ManageOwners/RemoveOwnerModal/screens/Review/index.tsx

```diff
@@ -201,7 +201,7 @@ export const ReviewRemoveOwnerModal = ({
                   </Paragraph>
                 </Row>
                 <Hairline />
-                <Row className={classes.selectedOwner}>
+                <Row className={classes.selectedOwner} data-testid="remove-owner-review">
```

Comment:
why do we need test ids there and cant use getByText with owner address?

Reply:
In this case we would like to ensure that the owner is set under the removal label, not at any place inside the modal, as if selected incorrectly we could still get a positive match to a generic "isPresent" but without being the selected to remove, as we are showing the full list of owners there